src/Breadcrumb/index.d.ts

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export interface BreadcrumbProps {
+  links: Array<{
+    label: string,
+  }>,
```

Comment:
Maybe we'll allow additional props to be passed to the link component?
I see on the Paragon documentation site the `href` prop for the array of links in the `Breadcrumb` component.

Reply:
👍